bin/setup

```diff
@@ -50,11 +50,20 @@ Dir.chdir APP_ROOT do
   run "cp pwned_passwords/pwned_passwords.txt.sample pwned_passwords/pwned_passwords.txt"
 
   puts "\n== Installing dependencies =="
+  brew_installed = system "brew -v 2>&1"
+  run "brew bundle" if brew_installed
   run "gem install bundler --conservative"
   run 'gem install foreman --conservative && gem update foreman'
   run "bundle check || bundle install --without deploy production"
   run "yarn install"
 
+  puts "\n== Stopping running services to ensure clean start =="
+  run "brew services stop --all"
```

Comment:
Do we need the condition here?

```suggestion
  run "brew services stop --all" if brew_installed
```

Reply:
I think that's to not blow up in CI where we run on linux and don't have homebrew installed?

Reply:
Good catch @aduth ! Suggestion committed.

@zachmargolis that's correct, and also for possible PC or Linux users setting the project up locally, though I'm not sure we have any.